grails-app/controllers/com/careerfly/organization/interview/InterviewController.groovy

```diff
@@ -111,70 +107,61 @@ class InterviewController {
             return
         }
 
+        City cityInstance = City.get(params.cityID)
+        cityInstance.name = params.companyAddress
+        cityInstance.save(flush: true)
+
+        Company companyInstance = Company.get(params.companyID)
+        companyInstance.name = params.companyName
+        companyInstance.save(flush: true)
 
-        println "here 1111"
```

Comment:
@Sagars09 It's good you cleaned up the `update` action but the code could have made DRY using services.